client/src/components/DownloadOption.js

```diff
@@ -42,6 +42,11 @@ export const DownloadOption = ({ computedFile, handleSelectFile }) => {
         </Box>
       </Box>
       <Box gridArea="body" margin={{ bottom: 'small' }}>
+        {isMergedObjectsAvailable && (
```

Comment:
This is the wrong check. Please check `computedFile.is_merged`, otherwise this warning will show when a user has not selected merged but a merged option is available.